hack/components/bump_knmstate.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source hack/components/common_functions.sh
+
+NMSTATE_URL=$(cat components.yaml | shyaml get-value components.nmstate.url)
```

Comment:
do we need to install shyaml ?

Reply:
yes. There are alternatives but I had no objections keeping it, as it's a dev tool doing a dev job..

Reply:
maybe then worth to verify it exists in the main script ?
or use a container of it

Reply:
you mean install it if it's not installed?

Reply:
i dont think people would like the script to install, either just warn + exit 1 before doing anything
or use a containerized version

Reply:
+1 on containerized

Also having some `get_component_url` and friends at common functions would be nice so we wrap the shyaml call at one place and we decide there how to run it.